sdk/python/setup.py

```diff
@@ -67,7 +66,7 @@
       'Topic :: Software Development :: Libraries',
       'Topic :: Software Development :: Libraries :: Python Modules',
     ],
-    python_requires='>=3.5',
```

Comment:
Is there a reason to bump the minimum supported version? Can we continue to support this version? Probably what we want here is:

```py
python_requires='>=3.5,<3.7'
```

Reply:
Echo Ajay's comment above. Is there any particular reason that the minimum required version is bumped to 3.6?

Reply:
Good point. Removing lgtm for now.

Reply:
> Is there a reason to bump the minimum supported version? Can we continue to support this version? Probably what we want here is:
> 
> ```python
> python_requires='>=3.5,<3.7'
> ```

No. We're fine with any higher version.

But 3.5 lacks many features, so it would make it easier to code some parts if we could move to 3.6+ only.